core/Azure.Mcp.Core/tests/Azure.Mcp.Tests/Client/CommandTestsBase.cs

```diff
@@ -16,27 +18,57 @@ public abstract class CommandTestsBase(ITestOutputHelper output) : IAsyncLifetim
     protected const string TenantNameReason = "Service principals cannot use TenantName for lookup";
 
     protected McpClient Client { get; private set; } = default!;
-    protected LiveTestSettings Settings { get; private set; } = default!;
+    public LiveTestSettings Settings { get; set; } = default!;
     protected StringBuilder FailureOutput { get; } = new();
     protected ITestOutputHelper Output { get; } = output;
 
-    private string[]? _customArguments;
+    public string[]? CustomArguments;
+    public TestMode TestMode = TestMode.Live;
```

Comment:
Wouldn't this be accessible through `Settings.TestMode`?

Reply:
I'm giving it a sane default. I don't really mind if it is immediately overriden.

Reply:
I meant the property in general is just a duplicated value. 

Settings sets / owns TestMode.  Then we immediately pull it out to a property beside Settings:
Settings = Deserialise<TestSettings>(json)
TestMode = Settings.TestMode

Then, for all uses, Settings.TestMode is in context,  but we look at TestMode instead.

If you're just adding a convenience accessor, I'd opt for an arrow property:
```suggestion
    public TestMode TestMode => Settings.TestMode;
```